src/sass/_theme_rst.sass

```diff
@@ -206,7 +206,7 @@
     @extend h2
 
   // This is the #href that shows up on hover. Sphinx's is terrible so I hack it away.
-  h1, h2, h3, h4, h5, h6, dl dt, p.caption, table > caption, .code-block-caption
+  h1, h2, h3, h4, h5, h6, dl dt, p.caption, table > caption, .code-block-caption, .eqno
```

Comment:
This CSS selector seems a bit too specific in the future we may want to use just `.headerlink`

Reply:
I'm guessing this list is just for applying the opacity on the parent element, otherwise we might be able to use `.headerlink`, yeah. This seems fine for now.

We could also use the entire `.math` parent element instead here, so that hovering over the math element will give the anchor link. I'm not sure how folks normally use the equation number here, but it might be hard to discover this anchor link if the hover event is just on the equation number.